bochs/configure

```diff
@@ -24299,6 +24137,40 @@ if test "$support_cet" -gt 0 -a "$use_x86_64" = 0; then
   as_fn_error $? "Control Flow Enforcement Technology feature emulation require x86-64 enabled" "$LINENO" 5
 fi
 
+support_uintr=0
+{ printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for User Level Interrupt support" >&5
+printf %s "checking for User Level Interrupt support... " >&6; }
```

Comment:
Did you regenerate ``configure`` script in 63fe1d4429f0f8e191f09b03c912c802a646f1cf or just edited it manually? I see suspicious leftovers of ``enable-cet``/``enable_cet`` in UINTR section (lines 24142-24146).

Reply:
Thanks, fixed